layouts/_partials/footer.html

```
@@ -0,0 +1,105 @@
+          {{ with .File }}
+          <a href="https://github.com/layer5io/docs/tree/master{{ strings.TrimPrefix hugo.WorkingDir .Filename }}" target="_blank"
+            rel="noreferrer">Edit This Page</a>
+          {{ end }}
+        </span>
+        <span>
+          <a class="status-link" href="https://layer5.statuspage.io" target="_blank" rel="noreferrer">
+            Layer5 Cloud Status <span class="pulse-icon"></span>
+          </a>
+        </span>
+      </span>
+      </span>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </span> ]

(tag-pair)